compiler-core/src/build.rs

```diff
@@ -296,10 +296,14 @@ pub enum Located<'a> {
     ModuleStatement(&'a TypedDefinition),
     FunctionBody(&'a TypedFunction),
     Arg(&'a TypedArg),
+    Annotation(&'a TypeAst, std::sync::Arc<type_::Type>),
```

Comment:
What do you think of just having the location here? It seems that's all we use and I can't think of anything else that we could take from this data structure that would be useful.

Reply:
Yeah I think replacing the TypeAst with just the location is reasonable. The type itself we need for figuring out the type constructor

Reply:
Sorry yes that's what I meant, in place of the AST